tests/unit/sinon-sandbox-with-global-access-test.js

```diff
@@ -66,6 +66,28 @@ test('using useFakeTimers API continues to work', function(assert) {
   restoreSandbox();
 });
 
+test('using useFakeXMLHttpRequest API continues to work', function(assert) {
```

Comment:
It would be nice to share tests that need to run in both global and non-global modes instead of duplicating. Seems fine for a future refactor though...

Reply:
I agree. I'll look into this in a subsequent PR.